server/lib/validation.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     INTEGER: joi.number().integer(),
     OFFSET: joi.number().integer().min(0),
     REFERRER: joi.string().max(2048).uri({ scheme: [ 'android-app', 'http', 'https' ]}).allow('none'),
+    RESUME: joi.string().regex(/^[\w]+={0,2}/), // match any alpha numberic string that could end with = or ==
```

Comment:
Is this definitely right?

`\w` is the alphanumerics plus `_`, which only makes 63 characters if `=` is reserved for padding. The base 64 encoding I'm familiar with is from [RFC 4648](https://tools.ietf.org/html/rfc4648#section-4), which uses the alphanumerics plus `+` and `/`, with `=` reserved for padding. But I know there are other base 64 encodings too, in the RFC itself there's one that uses `_` and `-` for the 63rd and 64th values.

Which one do we use?